themes/digital.gov/layouts/partials/core/featured-resource.html

```diff
@@ -0,0 +1,43 @@
+<!-- prettier-ignore-start -->
+{{/* Featured rounded box that displays a guide kicker, title, summary, and image and links to a resource
+
+  @param $resourcePath path to guide "/guides/dap"
+  @param $kicker display kicker text if exists
+
+  Example usage:
+  {{< featured-resource resourcePath = "/guides/dap/" kicker = "kicker to display">}}
+*/}}
+<!-- prettier-ignore-end -->
```

Comment:
Let's move this documentation comment to `shortcodes/featured-resource.html` for consistency.